app/controllers/state_file/intake_logins_controller.rb

```diff
@@ -32,8 +37,14 @@ def update
       end
     end
 
+    def account_locked; end
```

Comment:
Adds this page
<img width="672" alt="Screenshot 2024-01-25 at 12 30 29 PM" src="https://github.com/codeforamerica/vita-min/assets/49880002/73e72a4f-29ff-4ef4-8631-74f32a173957">